src/cli/sign.rs

```diff
@@ -19,12 +19,12 @@ use anyhow::{anyhow, bail, Context, Result};
 use camino::Utf8PathBuf;
 use clap::Args;
 use p384::ecdsa::signature::Signer as _;
-use p384::ecdsa::SigningKey;
+use p384::ecdsa::{Signature, SigningKey};
 use p384::elliptic_curve::sec1::Coordinates;
 use p384::pkcs8::DecodePrivateKey;
 use p384::EncodedPoint;
 use sgx::crypto::{rcrypto::*, *};
-use sgx::signature::{Author, Body, Signature};
+use sgx::signature::{Author, Body};
```

Comment:
Hmm.. you changed `Signature` and `unsafe impl ByteSized for Signature` does not apply anymore to `sgx::signature::Signature` then